src/components/ui/filecoinAddressAccept.tsx

```diff
@@ -9,20 +9,42 @@ const FileCoinAddressAccept: FC<{
 }> = ({ handleClick }) => {
   const [value, setValue] = useState<string>("");
   const [disabled, setDisabled] = useState<boolean>(true);
+  const [error, setError] = useState<string>("");
+
+  const validateAddress = (address: string): boolean => {
+    const filecoinAddressPattern = /^(f0|f1|f2|f3|f4)[a-zA-Z0-9]*$/;
```

Comment:
```suggestion
    const filecoinAddressPattern = /^f[0-4][a-zA-Z0-9]*$/;
```
Just a shorter alternative, completely not important.